mitodl/open-discussions
# add play button overlay for video cover images

#### Pre-Flight checklist


<img width="1111" alt="Screen Shot 2019-12-26 at 5 59 02 PM" src="https://user-images.githubusercontent.com/1934992/71493048-e5984600-2809-11ea-92f3-dadb7e6e6f5e.png">
<img width="441" alt="Screen Shot 2019-12-26 at 5 59 24 PM" src="https://user-images.githubusercontent.com/1934992/71493056-e9c46380-2809-11ea-9fbf-1f3e5c867c3b.png">
<img width="429" alt="Screen Shot 2019-12-26 at 5 59 28 PM" src="https://user-images.githubusercontent.com/1934992/71493057-ed57ea80-2809-11ea-9f7e-7638e5daa30f.png">
<img width="465" alt="Screen Shot 2019-12-26 at 6 00 24 PM" src="https://user-images.githubusercontent.com/1934992/71493060-f052db00-2809-11ea-8f00-7d9f9ea670dd.png">
<img width="729" alt="Screen Shot 2019-12-26 at 6 01 02 PM" src="https://user-images.githubusercontent.com/1934992/71493062-f47ef880-2809-11ea-883d-3b256b75a178.png">
<img width="1" alt="Screen Shot 2019-12-26 at 6 01 16 PM" src="https://user-images.githubusercontent.com/1934992/71493068-f8ab1600-2809-11ea-8b40-f5cf625bed6c.png">
<img width="767" alt="Screen Shot 2019-12-26 at 6 01 30 PM" src="https://user-images.githubusercontent.com/1934992/71493069-f8ab1600-2809-11ea-922b-b1624753e751.png">
<img width="1181" alt="Screen Shot 2019-12-26 at 6 01 39 PM" src="https://user-images.githubusercontent.com/1934992/71493070-f8ab1600-2809-11ea-99f1-59d63fc23828.png">
<img width="433" alt="Screen Shot 2019-12-26 at 6 01 49 PM" src="https://user-images.githubusercontent.com/1934992/71493071-f943ac80-2809-11ea-82fc-75f88d927e57.png">



#### What are the relevant tickets?
https://github.com/mitodl/open-discussions/pull/2512

#### What's this PR do?
Add a play button image overlay over video cover images in the /learn/search and /learn views

#### How should this be manually tested?
Run the app locally
Ensure that the /learn/search and /learn views have play buttons overlayed over video cover images and the overlay looks good on different screen sizes 

#### Where should the reviewer start?
(Optional)

#### Any background context you want to provide?
(Optional)

#### Screenshots (if appropriate)
(Optional)

#### What GIF best describes this PR or how it makes you feel?
(Optional)
